intro-Linux/ch_beyond_the_basics.tex

```diff
@@ -301,16 +309,15 @@ \section{Basic Shell Scripting}
 
 However there are some rules you need to abide by.
 
-Here is a link to a very detailed guide should you need more information:
-http://www.tldp.org/LDP/Bash-Beginners-Guide/html/
+Here is a \href{http://www.tldp.org/LDP/Bash-Beginners-Guide/html/}{very detailed guide} should you need more information.
 
 \subsection{Shebang}
-The first line of the script is the so called shebang (``\#'' is sometimes called
-hash and ``!'' is sometimes called bang). This line tells the shell which
+The first line of the script is the so called shebang (\verb|#| is sometimes called
+hash and \verb|!| is sometimes called bang). This line tells the shell which
 command should execute the script. In the most cases this will simply be the
 shell itself. The line itself looks a bit weird, but you can copy paste this
 line as you need not worry about it further. It is however very important this
-is the very first line of the script!
+is the very first line of the script! These are all valid shebangs:
```

Comment:
Maybe mention that only *one* of these should be used?

Also make them separate boxes to clarify that?